### ISSUE TITLE
TASK_NAME/validation.jsonl used for tuning hyperparameters for zero-shot testing?

### ISSUE THREAD

**Issue by tomyoung903** [THUDM/GLM-130B#64]
Hi, thanks for the great work!

I'm curious about how the provided validation set may affect how zero-shot testing is conducted. 

Specifically, is the TASK_NAME/validation.jsonl (accompanying TASK_NAME/X_test.jsonl) used for tuning certain hyperparameters at test time?
